specification/synapse/data-plane/Microsoft.Synapse/preview/2020-08-01-preview/checkAccessSynapseRbac.json

```diff
@@ -170,9 +170,15 @@
     },
     "CheckPrincipalAccessResponse": {
       "description": "Check access response details",
-      "type": "array",
-      "items": {
-        "$ref": "#/definitions/CheckAccessDecision"
+      "type": "object",
+      "properties": {
+        "AccessDecisions": {
+          "description": "Check access response details",

```

Comment:
Are we duplicating the description here with line 172? Seems like they are two different things.